src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list_header.html.twig

```diff
@@ -46,8 +46,8 @@
         {{ 'Edit %combinations_number% combinations'|trans({'%s': ''}, 'Admin.Catalog.Feature') }}
       </button>
       {% if isMultistoreActive %}
-        {% set deleteLabel = 'Delete %combinations_number% combinations for shop "%shop%"'|trans({'%shop%': contextShopName}, 'Admin.Catalog.Feature') %}
-        {% set modalMessage = 'Are you sure you want to delete selected combinations for shop "%shop%"?'|trans({'%shop%': contextShopName}, 'Admin.Catalog.Notification') %}
+        {% set deleteLabel = 'Delete %combinations_number% combinations from store "%shop%"'|trans({'%shop%': shopName}, 'Admin.Catalog.Feature') %}
```

Comment:
```suggestion
        {% set deleteLabel = 'Delete %combinations_number% combinations from store "%store_name%"'|trans({'%store_name%': shopName}, 'Admin.Catalog.Feature') %}
```
Just a suggestion but maybe the placeholder should follow the same convention of using store and not shop